.semaphore/semaphore.yml

```diff
@@ -78,7 +78,7 @@ blocks:
               value: extensions-contrib/confluent-extensions
           commands: &run_tests
             - >
-              MAVEN_OPTS="${MAVEN_OPTS} -Xmx800m" ${MVN} test -pl ${MAVEN_PROJECTS}
+              MAVEN_OPTS="${MAVEN_OPTS} -Xmx1g" ${MVN} test -pl ${MAVEN_PROJECTS}
```

Comment:
maybe let's separate this out, to make it simpler to cherry-pick your change and submit it upstream.

Reply:
Yeah I'll do that.